### PR TITLE
Voidcrewing shiptest name replacement

### PR DESCRIPTION
**Pull request by HourlyLimit**
## About The Pull Request

Just renaming stuff (a lot of stuff) I have read it over it should all be fine you can read it over as well

## Why It's Good For The Game

So we don't need to rename stuff later and it makes it show we can be our own thing
**also why its good: ShitPest gon :)** _- Bread Man (John Bob)_
## Changelog
:cl:
add: Added Lots of renames
/:cl:
image

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
me when me wheni when i find replace functi on

**Comment by HourlyLimit** [comment]
> me when me wheni when i find replace functi on

Indeed

File: code/datums/ruins/icemoon.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	description = "Blown up by an unfortunate accident."
 	suffix = "icemoon_surface_engioutpost.dmm"
 
-/datum/map_template/ruin/icemoon/slimerancher //Shiptest edit
```
Comment (GuiltyNeko) [inline code/datums/ruins/icemoon.dm:16]: I feel like it's incredibly unnecessary to wipe shiptest out of comments, that removes a significant amount of context from "why was this decision made?"

File: code/datums/ruins/space.dm [hunk 2]
```diff
@@ -177,7 +177,7 @@
 	description = "Until very recently this pirate radio station was used to harangue local space stations over a variety of perceived \"ethics violations\". \
 	It seems like someone finally got sick of it, but the equipment still works."
 
-/* shiptest: derelict is too big. we're commenting it out for now
+/* voidcrew: derelict is too big. we're commenting it out for now
```
Comment (GuiltyNeko) [inline code/datums/ruins/space.dm:180]: In this instance, renaming shiptest to voidcrew implies we made this decision, but we didn't. We shouldn't blame ourselves for actions that are not ours.

File: config/config.txt [hunk 3]
```diff
@@ -10,17 +10,17 @@ $include atmos_mix.txt
...
+# @SERVERNAME voidcrew
 # Which sets the SERVERNAME, and disallows admins from being able to change it using View Variables.
 # @LOG_TWITTER 0
 # Which explicitly disables LOG_TWITTER, as well as locking it.
 # There are various options which are hard-locked for security reasons.
 
-## Server name: This appears at the top of the screen in-game and in the BYOND hub. Uncomment and replace 'Shiptest' with the name of your choice.
-# SERVERNAME Shiptest
+## Server name: This appears at the top of the screen in-game and in the BYOND hub. Uncomment and replace 'voidcrew' with the name of your choice.
+# SERVERNAME voidcrew
```
Comment (GuiltyNeko) [inline config/config.txt:20]: If you're going to take this seriously, capitalize voidcrew where shiptest was capitalized.

File: html/changelogs/.all_changelog.yml [hunk 4]
```diff
@@ -40943,9 +40943,9 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
 2021-11-04:
   Bokkiewokkie:
   - bugfix: Hopefully made cloning no longer make monochromacy temporary.
-  - bugfix: Shiptest maintenance areas are now no longer named "Hallway".
+  - bugfix: voidcrew maintenance areas are now no longer named "Hallway".
   - bugfix: Allows you to access Syndicate sleepers from the inside again.
-  - code_imp: The dream maker enviroment has been renamed to shiptest.
+  - code_imp: The dream maker enviroment has been renamed to voidcrew.
```
Comment (GuiltyNeko) [inline html/changelogs/.all_changelog.yml:40948]: Absolutely do not rename shiptest to voidcrew in the changelogs, that's as rude as you get.

**Review by SuperNovaa41 — CHANGES_REQUESTED** [review]
Why did you delete `.vscode/settings.json` please do not do this.

Don't touch the changelog, don't change the comments that are `shiptest edit` or whatever else they may be. Those weren't us I'd rather have those documented

**Comment by HourlyLimit** [comment]
eh it was a joke pull anyways

**Pull request closed without merging**